src/cider/nrepl/middleware/format.clj

```diff
@@ -22,15 +23,22 @@
      (assoc acc
             (keyword->symbol kw)
             (walk/postwalk #(cond-> % (string? %) keyword) rule)))
-   fmt/default-indents
```

Comment:
Removed this, it was preventing regexes from working (e.g. the default `let` would take precedence over a custom `let.*`)

cljfmt doesn't need its defaults to be passed again, probably by doing so one only creates ambiguity